src/modules/getvshow.js

```diff
@@ -1,5 +1,26 @@
+/* eslint-disable import/no-cycle */
+import renderShows from './render.js';
+
 const getvShow = async () => {
   const response = await fetch('https://api.tvmaze.com/shows');
   return response.json();
 };
-export default getvShow;
\ No newline at end of file
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/ED1rQK1snOoAMYBCYLLf/likes';
+const addLikes = async (id) => {
+  await fetch(url,
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        item_id: id,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+  renderShows();
+};
+const getlikes = (async () => {
+  const response = await fetch(url, { method: 'GET' });
+  return response.json();
```

Comment:
The default method for fetch is GET, so there's no need to specify it again in the function